app/src/main/java/com/xa/xpensauditor/AddTransactionActivity.java

```diff
@@ -54,10 +54,10 @@
 
 public class AddTransactionActivity extends AppCompatActivity {
 
-    private Firebase mRootRef;
-    private Firebase RefUid;
-    private Firebase RefTran1, RefCatSum1, RefCat, UnCatTran;
-    private String Tid;
+//    private Firebase mRootRef;
+//    private Firebase RefUid;
+//    private Firebase RefTran1, RefCatSum1, RefCat, UnCatTran;
+//    private String Tid;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 0, expected is 4, indentation should be the same level as line 61.